lib/rqrcode/qrcode/qrcode.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+
+module RQRCode #:nodoc:
+  class QRCode
+    extend Forwardable
+    def_delegators :@qrcode, :to_s
```

Comment:
Is there any reason to prefer this over `delegate :to_s, to: @qrcode`? (Not disputing it, just wondering!)

Reply:
Isn't `delegate` something Rails provides?

Reply:
Ah yep, you're absolutely right 👍